src/arviz_plots/plots/compareplot.py

```diff
@@ -0,0 +1,137 @@
+    figsize : (float, float), optional
+        If `None`, size is (10, num of models) inches.
+    target : bokeh figure, matplotlib axes, or plotly figure optional
+    backend : {"bokeh", "matplotlib", "plotly"}
+        Select plotting backend. Defaults to rcParams["plot.backend"].
+
+    Returns
+    -------
+    axes :bokeh figure, matplotlib axes or plotly figure
+
+    See Also
+    --------
+    plot_elpd : Plot pointwise elpd differences between two or more models.
+    compare : Compare models based on PSIS-LOO loo or WAIC waic cross-validation.
+    loo : Compute Pareto-smoothed importance sampling leave-one-out cross-validation (PSIS-LOO-CV).
+    waic : Compute the widely applicable information criterion.
+
+    References
+    ----------
+    .. [1] Vehtari et al. (2016). Practical Bayesian model evaluation using leave-one-out
+       cross-validation and WAIC https://arxiv.org/abs/1507.04544
+    """
+    information_criterion = ["elpd_loo", "elpd_waic"]
+    column_index = [c.lower() for c in cmp_df.columns]
+    for i_c in information_criterion:
+        if i_c in column_index:
+            break
+    else:
+        raise ValueError(
+            "cmp_df must contain one of the following "
+            f"information criterion: {information_criterion}"
+        )
+
+    if backend is None:
+        backend = rcParams["plot.backend"]
+
+    if relative_scale:
+        cmp_df = cmp_df.copy()
+        cmp_df[i_c] = cmp_df[i_c] - cmp_df[i_c].iloc[0]
+
+    if figsize is None:
+        figsize = (10, len(cmp_df))
+
+    p_be = import_module(f"arviz_plots.backend.{backend}")
+    _, target = p_be.create_plotting_grid(1, figsize=figsize)
+    linestyle = p_be.get_default_aes("linestyle", 2, {})[-1]
+
+    # Compute positions of yticks
+    yticks_pos = list(range(len(cmp_df), 0, -1))
+
+    # Get scale and adjust it if necessary
+    scale = cmp_df["scale"].iloc[0]
+    if scale == "negative_log":
+        scale = "-log"
+
+    # Compute values for standard error bars
+    se_list = list(zip((cmp_df[i_c] - cmp_df["se"]), (cmp_df[i_c] + cmp_df["se"])))
+
+    # Plot ELPD point statimes
+    p_be.scatter(cmp_df[i_c], yticks_pos, target, color=color)
+    # Plot ELPD standard error bars
+    for se_vals, ytick in zip(se_list, yticks_pos):
+        p_be.line(se_vals, (ytick, ytick), target, color=color)
+
+    # Add reference line for the best model
+    p_be.line(
+        (cmp_df[i_c].iloc[0], cmp_df[i_c].iloc[0]),
+        (yticks_pos[0], yticks_pos[-1]),
+        target,
+        color=color,
+        linestyle=linestyle,
+        alpha=0.5,
+    )
+
+    # Add band for statistically undistinguishable models
+    if similar_band:
+        if scale == "log":
+            x_0, x_1 = cmp_df[i_c].iloc[0] - 4, cmp_df[i_c].iloc[0]
+        else:
+            x_0, x_1 = cmp_df[i_c].iloc[0], cmp_df[i_c].iloc[0] + 4
+
+        p_be.fill_between_y(
+            x=[x_0, x_1],
+            y_bottom=yticks_pos[-1],
+            y_top=yticks_pos[0],
```

Comment:
I would make this cover a bit more than the exact y range where data is plotted.